Swashbuckle.AspNetCore.sln

```diff
@@ -1,4 +1,3 @@
-﻿
```

Comment:
I'm guessing this was an invisible file-encoding character change, but can you revert it from the PR so the file is completed un-edited please?